### ISSUE TITLE
on java gui not speak selected item

### ISSUE THREAD

**Issue by limaoscar** [nvaccess/nvda#11735]
the nvda 2020.X until actual beta4 of 2020.3 does not work as previous nvda 2019.x python2 version,when  surfing java gui, before, I can click on items in the gui and nvda speack the name of item, now it does not pseak anything.
to let it speak ,but only one time , I have to select another part of gui and after back to select main window elemnts.
In attach the log where appears errors 

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Speaking ['wx._core.wxAssertionError: C++ assertion "node" failed at ..\\..\\src\\msw\\menu.cpp(890) in wxMenu::DoRemove(): bug in wxMenu::Remove logic\r']
[20201006_nvda.log](https://github.com/nvaccess/nvda/files/5333137/20201006_nvda.log)

**Comment by limaoscar** [comment]
hello I forgot to write some details:
Steps to reproduce: open java gui with graphich labeled elemnt and try to select them, the scrren nreader does not speack the name of label

Actual behavior:  no speack of label name in java gui graphical elemnts.
Expected behavior: using nvda 2019.2 it speaks the label name of grapthcal elemnts on gui java.


System configuration
NVDA installed/portable/running from source: installaed 
NVDA version: 2020.3 b4
Windows version: 10 2004
Name and version of other software in use when reproducing the issue: java web start gui
Other information about your system:
Other questions
Does the issue still occur after restarting your computer?
Have you tried any other versions of NVDA? If so, please report their behaviors.
If addons are disabled, is your problem still occuring?
Did you try to run the COM registry fixing tool in NVDA menu

**Comment by leonardder** [comment]
To verify, did this stop working on 2019.3?

Would you be able to provide a log file for the moments you perform the steps to reproduce?

**Comment by limaoscar** [comment]
I can add another information, I found that the focus of nvda does not go on the mouse click , I mean that  in 2019 version I click an item on java gui and the focus  and cursor of nvda goes on that point, and the nvda speak the information about that object, in 2020 version when  I click the item , i see the focus  and the item is selected, but the nvda focus remain on toolbar of java gui. I have to click on another part of gui to and after select the item on main window to let nvda move the focus of cursor on the item. Hope it can clarify little bit, sorry but there are no other investigation  I can do .

**Comment by limaoscar** [comment]
I have a 2019 portable, where can I find the logs?
